src/uu/tee/src/tee.rs

```diff
@@ -89,6 +89,16 @@ pub fn uu_app() -> Command {
         .override_usage(format_usage(USAGE))
         .after_help(AFTER_HELP)
         .infer_long_args(true)
+        // Since we use value-specific help texts for "--output-error", clap's "short help" and "long help" differ.
```

Comment:
I'm not sure about this change. It's more compatible, sure, but is it better? I want to limit all sorts of edge cases in the code and might prefer disabling that test here. Or can we force the bit for `--output-error` to always be shown?

Note that this will be fixed by uutils-args too by the way, so maybe it just doesn't matter much either way.

Reply:
I don't quite understand? All that this piece of code does is force the bit for `--output-error` to always be shown, just like you suggest.